lib/Config.js

```diff
@@ -106,14 +106,6 @@ class Config extends EventEmitter {
                 { host: '127.0.0.1', port: 6379 });
         }
 
-        if (parsedConfig.localCache) {
```

Comment:
Should be removed from the json file as well, maybe also from the operator (https://github.com/scality/zenko-operator/blob/development/1.6/pkg/controller/zenko/reconcile_backbeat_config.go#L982)

Reply:
done here : https://github.com/scality/backbeat/pull/2534/commits/1ad8a8219e5107e9c1340432457538387a5833ec#diff-10001999fea4d2498276120abf7902b81510153910d8314c7493ddafa6e19e09L324-L331, for the operator i'm going to check then create a ticket if necessary

Reply:
ticket created here : https://scality.atlassian.net/browse/ZKOP-408